src/extensions/core/editAttention.ts

```diff
@@ -50,12 +57,12 @@ app.registerExtension({
         if (text[end] === ')') closeCount++
         end++
       }
-      if (end === text.length) return false
+      if (end === text.length) return null
```

Comment:
I thought it would make more sense to use a Nullable type here, instead of Enclosure | boolean. 
It doesn't impact anything since the only place where it is used is an if and null is falsy:

```ts
const nearestEnclosure = findNearestEnclosure(inputField.value, start)
if (nearestEnclosure) {
     ...
} else {
     ...
}
```